samples/AspNetCore/SecretStoreConfigurationProviderSample/Startup.cs

```diff
@@ -0,0 +1,69 @@
+namespace SecretStoreConfigurationProviderSample
+{
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// Startup class.
+    /// </summary>
+    public class Startup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Startup"/> class.
+        /// </summary>
+        /// <param name="configuration">Configuration.</param>
+        public Startup(IConfiguration configuration)
+        {
+            this.Configuration = configuration;
+        }
+
+        /// <summary>
+        /// Gets the configuration.
+        /// </summary>
+        public IConfiguration Configuration { get; }
+
+        /// <summary>
+        /// Configures Services.
+        /// </summary>
+        /// <param name="services">Service Collection.</param>
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddHttpClient();
```

Comment:
Looks good overall, few more questions: where is this httclient used?

Reply:
Removed.